src/Client.ts

```diff
@@ -3,27 +3,27 @@ import HttpClient from './Interfaces/HttpClient';
 import Auth from './Resources/Auth/Auth';
 import Users from './Resources/Users/Users';
 import UserSubscriptions from './Resources/UserSubscriptions/UserSubscriptions';
-import Basket from './Resources/Basket/Basket';
-import BasketLine from './Resources/Basket/BasketLine';
 import Subscriptions from './Resources/Subscriptions/Subscriptions';
 import SubscriptionAddons from './Resources/SubscriptionAddons/SubscriptionAddons';
 import Codes from './Resources/Codes/Codes';
 import Toybox from './Resources/Toybox/Toybox';
 import { loadWaccResources, WaccResources } from './Resources/Wacc';
+import { loadGuestResources, GuestResources } from './Resources/Guest';
+import { loadUserResources, UserResources } from './Resources/User';
 import Products from './Resources/Products/Products';
 
 export default class Client {
     public auth!: Auth;
     public users!: Users;
     public userSubscriptions!: UserSubscriptions;
-    public basket!: Basket;
-    public basketLine!: BasketLine;
     public subscriptions!: Subscriptions;
     public subscriptionAddons!: SubscriptionAddons;
     public codes!: Codes;
     public toybox!: Toybox;
     public products!: Products;
     public wacc!: WaccResources;
+    public guest!: GuestResources;
+    public user!: UserResources;
```

Comment:
I'm wondering if we need to move all of the user's stuff under the user endpoint? 

whirli.auth
whirli.subscriptions
whirli.subscriptions.addons
whirli.codes
whirli.products
whirli.wacc.*
whirli.guest.*
whirli.users
whirli.users.subscriptions
whirli.users.baskets
whirli.users.baskets.lines
whirli.users.toybox
whirli.users.details

What do you think?